spec/models/item_spec.rb

```diff
@@ -77,22 +77,14 @@
     end
 
     describe '#invoice_item_status' do
-      xit 'should give the quantity of the item sold' do
-        invoice_item = InvoiceItem.first
-        invoice = Invoice.find_by(id: invoice_item.invoice_id)
-        item = Item.find_by(id: invoice_item.item_id)
-
-        expect(item.invoice_item_status(invoice)).to eq("packaged")
+      it 'should give the quantity of the item sold' do
```

Comment:
this describe block is a little confusing. it say that it 'should give the quantity of the item sold, but it gives the status

Reply:
looks like it might be a copy paste error